data/selinux/snappy.te

```diff
@@ -156,6 +159,10 @@ systemd_config_all_services(snappy_t)
 systemd_manage_all_unit_files(snappy_t)
 systemd_manage_all_unit_lnk_files(snappy_t)
 systemd_exec_systemctl(snappy_t)
+systemd_reload_all_services(snappy_t)
```

Comment:
Should we also have `systemd_start_all_services`?

Reply:
From inspecting the policy, this is likely covered by systemd_config_all_services(). This is what sesearch tells me:
```
$ sesearch -A -c service -s snappy_t
Found 2 semantic av rules:
   allow snappy_t init_script_file_type : service { start stop status reload enable disable kill load } ; 
   allow snappy_t systemd_unit_file_type : service { start stop status reload enable disable kill load } ; 
```